ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.16.yaml

```diff
@@ -398,6 +398,21 @@ tests:
       NODE_TUNING: "true"
       TEST_TYPE: conformance-parallel
     workflow: openshift-e2e-libvirt-upi
+- as: ocp-e2e-ovn-agent-remote-libvirt-s390x
+  cluster: build01
+  cron: 0 4 * * 1-5
+  steps:
+    cluster_profile: libvirt-s390x
+    dependencies:
+      OPENSHIFT_INSTALL_TARGET: release:s390x-latest
```

Comment:
Just curious, did we not want to use the multi-payload for these agent jobs?

Reply:
There's a bug with agent-based installs that breaks when using the multi-payload.
I've started investigating to see if it's still an issue.